DEVELOPMENT.md

```diff
@@ -181,6 +158,29 @@ to explicitly enable it.
 ko apply -f test/config/sugar.yaml
 ```
 
+## Starting Eventing Controller
```

Comment:
The channels (IMC channels) as well as the broker are optional, and the eventing controller should be installed first. Not sure why exactly you want to move this below the channels?

Here is how I usually install the bits:

```
ko apply -f config && ko apply -Rf config/channels/in-memory-channel/ && ko apply -f config/brokers/mt-channel-broker 
```